downstream/titles/aap-installation-guide/master.adoc

```diff
@@ -14,6 +14,7 @@ Thank you for your interest in {PlatformName}. {PlatformNameShort} is a commerci
 
 This guide helps you to understand the installation requirements and processes behind installing {PlatformNameShort}. This document has been updated to include information for the latest release of {PlatformNameShort}.
  
+include::../../../common/making-open-source-more-inclusive.adoc[]
```

Comment:
Should create a symlink to the common folder in each directory so the include directive is like this: `include::common/making-open-source-more-inclusive.adoc[]`

The publishing system doesn't play nicely with `../` (although that might have changed).

Reply:
@dcdacosta Give me a shout if you want me to create the symlinks. I don't mind doing it. Also what do you think about changing `common` to `aap-common`?